package.json

```diff
@@ -19,10 +19,14 @@
   },
   "dependencies": {
     "@react-native-community/slider": "^2.0.8",
+    "@react-native-firebase/app": "^6.4.0",
```

Comment:
We use v5 of firebase library already. We shouldn't mix it with v6

the best option would be to migrate to v6 ;)